demos/custom/src/pages/index.tsx

```diff
@@ -36,7 +36,7 @@ const HomePage = () => {
       )}
 
       {isLoggedIn ? (
-        <Button onClick={AuthService.logout}>Logout</Button>
+        <Button onClick={() => AuthService.logout()}>Logout</Button>
```

Comment:
Does this need to be changed? If not, can we revert this?

Reply:
The problem here was that `AuthService.logout` was being passed the React event directly from `onClick`, instead of  the optional `LogoutOptions`, and TypeScript was complaining.
We could maybe specify the logout parameter to be `opts?: auth0.LogoutOptions | React.MouseEvent< HTMLButtonElement>` but it seems like a bit of a hack just to avoid the arrow function there? Let me know your thoughts.